tico/passes/remove_redundant_reshape.py

```diff
@@ -348,7 +348,7 @@ def call(self, exported_program: ExportedProgram) -> PassResult:
             assert isinstance(reshape1_input, torch.fx.Node), type(reshape1_input)
             assert isinstance(size, list), type(size)
             for s in size:
-                assert isinstance(s, int), type(s)
+                assert isinstance(s, (int, torch.SymInt)), type(s)
```

Comment:
Most of the current passes are written without consideration of dynamic shapes. We may need more tests to support dynamic shapes in existing passes.

Reply:
Ah, current draft is just a quick patch for checking a feasibility of #28. I have a plan that introduces a dynamic shape tests using onert backend internally.

Reply:
Just curiosity. Does onert run well on x86?

Reply:
> Just curiosity. Does onert run well on x86?

Yes, onert runs well.